build/publish.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
```

Comment:
would be good to add a README file to the build/ directory to explain the approach